doc/libstemmer_python_README

```diff
@@ -0,0 +1,84 @@
+Snowball stemming library collection for Python
+===============================================
+
+How to use library
+------------------
+
+The ``snowballstemmer`` module has two functions.
+
+The ``snowballstemmer.algorithms`` function returns a list of avilable algorithm name' string.
+
+The ``snowballstemmer.stemmer`` function accepts algorithm name and returns ``Stemmer`` objects.
+
+``Stemmer`` objects have ``Stemmer.stemWord(word)`` method and ``Stemmer.stemWords(word[])`` method.
+
+.. code-block:: python
+
+   import snowballstemmer
+
+   stemmer = snowballstemmer.stemmer('english');
+   print(stemmer.stemWords("We are the world".split()));
+
+``Stemmer`` objects have ``Stemmer.maxCacheSize`` property. They cache result within the value. Default is ``10000``.
+
+Accelerates Stemming
+--------------------
+
+if **PyStemmer** is installed, ``snowballstemmer.stemmer`` returns ``PyStemmer``\ 's ``Stemmer`` objects. This ``Stemmer`` object has same methods (``Stemmer.stemWord()``, ``Stemmer.stemWords()``).
+
+**PyStemmer** is a Snowball's ``libstemmer_c`` wrapper module and it returns 100% compatible result with **snowballstemmer**.
+
+**PyStemmer** has faster speed because it uses C-lang module, and **snowballstemmer** has higher usability because it is pure Python module.
+
+* `PyStemmer <http://pypi.python.org/pypi/PyStemmer/>`_
+
+Benchmark
+~~~~~~~~~
+
+Test Case: Snowball stemmer check data (16 algorithms, total 582560 words, cache hit 0%)
+Computer: MacBook Pro 3rd Gen Corei7 2.3GHz
+
+* Python 2.7 + **snowballstemmer** : 2m 30s
+* PyPy 1.9 + **snowballstemmer** : 45s
+* Python 2.7 + **PyStemmer** : 5s
+
+This test case is much harder than usual usecases!
+
+The TestApp example
+-------------------
+
+The ``testapp.py`` example program allows you to run any of the stemmers
+on a sample vocabulary.
+
+Usage::
+
+   testapp.py <algorithm> "sentences ... "
+
+.. code-block:: bash
+
+   $ python testapp.py English "sentences... "
+
+License
+-------
+
+It is a BSD licensed library.
+
+Copyright (c) 2013, Yoshiki Shibukawa
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification, are permitted provided
+that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright notice, this list of conditions and
+  the following disclaimer.
+* Redistributions in binary form must reproduce the above copyright notice, this list of conditions
+  and the following disclaimer in the documentation and/or other materials provided with the distribution.
```

Comment:
I believe BSD 3-clause is the correct licence here - for the gory details see:
http://article.gmane.org/gmane.comp.search.snowball/1558

I'm guessing @shibukawa just took this from the linked OSI page after OSI had unhelpfully changed the content...